src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -410,4 +415,87 @@ private MobileAdsDataProvider() {
         {"adtest", "SyntheticTests/Async/Hop/ExtraMarker", "mobile", "\"test-marker\":\"42\""}
     };
   }
+
+  @DataProvider
+  public static Object[][] adsSlotSizeMercury() {
+    return new Object[][]{
+        {
+            new Page("adtest", "SyntheticTests/MobileLeaderboard"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.MOBILETOP_LB)
+                .put("slotSize", new Dimension(320, 100))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "slotSize" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)